src/encode/pattern/mod.rs

```diff
@@ -531,6 +538,7 @@ enum FormattedChunk {
     Line,
     Thread,
     Target,
+    PID,
```

Comment:
The naming convention for acronyms in Rust is to only capitalize the first letter, so this would be `Pid` rather than `PID`.